src/PKSim.Presentation/DTO/Protocols/SchemaItemTargetDTO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using PKSim.Assets;
+
+namespace PKSim.Presentation.DTO.Protocols
+{
+   public class SchemaItemTargetDTO
```

Comment:
This is the new object representing either organ or compartment. I need an object to be able to bind to the grid view